src/main/webapp/WEB-INF/web.xml

```diff
@@ -17,4 +17,10 @@
         <location>/page-not-found</location>

```

Comment:
Downside is that local Tomcat has to be restarted for JSP/CSS changes etc, since it uses src's web.xml by default, but this will do for now.